compiler/rustc_target/src/target_features.rs

```diff
@@ -502,6 +503,7 @@ static RISCV_FEATURES: &[(&str, Stability, ImpliedFeatures)] = &[
     ),
     ("m", Stable, &[]),
     ("relax", Unstable(sym::riscv_target_feature), &[]),
+    ("supm", Unstable(sym::riscv_target_feature), &[]),
```

Comment:
As a general rule I don't think we should be exposing supervisor-level features. I don't think Supm even exposes anything to userspace directly, only that "some mechanism" exists for controlling pointer masking.

Reply:
Okay.
If that was a regular `Su` extensions (user-mode side of supervisor-related extension), I've pushed it but not this `Supm`.  It's too OS-dependent and I understand that this is not suitable for Rust target features.
Removing on the next version.